tests/load/test_update_utils.py

```diff
@@ -1,5 +1,274 @@
+    ],
+)
+def test_get_language_from_iso_positive(iso_code, language):
+    assert utils.get_language_from_iso(iso_code) == language
+
+
+def test_get_language_from_iso_negative():
+    with pytest.raises(ValueError) as excp:
+        _ = utils.get_language_from_iso("ixi")
+
+    assert str(excp.value) == "IXI is currently not a supported ISO language."
+
+
+@pytest.mark.parametrize(
+    "language, remove_words",
+    [
+        (
+            "english",
+            [
+                "of",
+                "the",
+                "The",
+                "and",
+            ],
+        ),
+        (
+            "french",
+            [
+                "of",
+                "the",
+                "The",
+                "and",
+            ],
+        ),
+        ("german", ["of", "the", "The", "and", "NeinJa", "et", "redirect"]),
+        ("italian", ["of", "the", "The", "and", "text", "from"]),
+        ("portuguese", ["of", "the", "The", "and", "jbutadptflora"]),
+        (
+            "russian",
+            [
+                "of",
+                "the",
+                "The",
+                "and",
+            ],
+        ),
+        ("spanish", ["of", "the", "The", "and"]),
+        ("swedish", ["of", "the", "The", "and", "Checklist", "Catalogue"]),
+    ],
+)
+def test_get_language_words_to_remove(language, remove_words):
+    test_case = unittest.TestCase()
+
+    # ignore order, only content matters
+    test_case.assertCountEqual(
+        utils.get_language_words_to_remove(language), remove_words
+    )
+
+
+def test_get_language_words_to_remove_negative():
+    with pytest.raises(ValueError) as excp:
+        _ = utils.get_language_words_to_remove("python")
+
+    assert str(excp.value) == "Python is currently not a supported language."
+
+
+@pytest.mark.parametrize(
+    "language, ignore_words",
+    [
+        (
+            "french",
+            [
+                "XXe",
+            ],
+        ),
+        ("german", ["Gemeinde", "Familienname"]),
+        ("italian", ["The", "ATP"]),
+        ("portuguese", []),
+        ("russian", []),
+        ("spanish", []),
+        ("swedish", ["databasdump"]),
+    ],
+)
+def test_get_language_words_to_ignore(language, ignore_words):
+    test_case = unittest.TestCase()
+
+    # ignore order, only content matters
+    test_case.assertCountEqual(
+        utils.get_language_words_to_ignore(language), ignore_words
+    )
+
+
+def test_get_language_words_to_ignore_negative():
+    with pytest.raises(ValueError) as excp:
+        _ = utils.get_language_words_to_ignore("JAVA")
+
+    assert str(excp.value) == "Java is currently not a supported language."
+
+
+def test_get_path_from_format_file():
+    assert utils.get_path_from_format_file() == "../../../../../.."
+
+
+def test_get_path_from_load_dir():
+    assert utils.get_path_from_load_dir() == "../../../.."
+
+
+def test_get_path_from_et_dir():
+    # TODO: file path is same as above. Is this correct?
```

Comment:
Yes it is. I think at one point they were at different levels, but they're in the same directory now. We could change this to `get_path_from_scribe_data_child_dir` and replace both?